engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/plan/NadelExecutionPlanFactory.kt

```diff
@@ -89,6 +90,7 @@ internal class NadelExecutionPlanFactory(
                     NadelHydrationTransform(engine),
                     NadelBatchHydrationTransform(engine),
                     NadelRenameTransform(),
+                    NadelRenameArgumentInputTypesTransform(),
```

Comment:
I think this needs to be above `NadelRenameTransform`

Reply:
yeah so what informs our ordering?  Is it just inherent knowledge of the code or is there some way to encode he ordering?

Reply:
I have moved it up one

Reply:
Yea so basically fields get fed through the transforms until there's no more transforms or the field is deleted/nulled. The way the rename transform works is that given

```
type User {
  id: ID! @renamed(from: "ari")
}
```

1. Rename transform deletes the `id` field
2. Adds an artificial field `rename__some_unique_alias: ari` based on a copy of the original `id` field (retains args etc.)
3. Copies `ari` into `id`

The issue is that step 1 deletes the field so if you place the argument transform after the rename transform, it won't run. That's a reason the rename transform is placed dead last 😄.